From: Ben Hutchings <ben@decadent.org.uk>
Date: Mon, 29 Jul 2013 00:05:51 +0100
Subject: [sparc] cpufreq: Convince genksyms that the ABI didn't change
Forwarded: not-needed

genksyms is really stupid: un-including a header whose definitions
didn't appear in the ABI stil changes symbol versions.  So restore the
#include <linux/cpu.h> for symbol version purposes.

(This only seems to affect SPARC; on other architectures
<linux/cpu.h> may still be indirectly included.)

--- a/drivers/cpufreq/cpufreq_governor.c
+++ b/drivers/cpufreq/cpufreq_governor.c
@@ -26,6 +26,9 @@
 #include <linux/tick.h>
 #include <linux/types.h>
 #include <linux/workqueue.h>
+#ifdef __GENKSYMS__
+#include <linux/cpu.h>
+#endif
 
 #include "cpufreq_governor.h"
 
